### DISCUSSION TITLE
💡Add phase voltages to mqtt topics

### DISCUSSION THREAD

**Discussion by RichieB2B** [dsmrreader/dsmr-reader#1501] [Questions & answers / How-to's] [answered]
### Feature

Currently phase_voltage_l1 / phase_power_current_l1 (and l2/l3) aka INSTANTANEOUS_VOLTAGE_L1 / INSTANTANEOUS_CURRENT_L1 in DSMR are missing from the mqtt topics. I need this for a custom load balancer I am writing. 

I can probably make a PR to add them.

**Comment by RichieB2B** [comment, marked as answer]
Never mind, https://github.com/dsmrreader/dsmr-reader/blob/v4/dsmr_mqtt/models/settings/telegram.py already includes the  phases and voltages. Just somehow my mapping did not include them even though I had never touched mqtt on dsmr-reader before.

**Comment by dennissiemensma** [comment]
It was added March last year: https://github.com/dsmrreader/dsmr-reader/blame/v4/dsmr_mqtt/models/settings/telegram.py#L64-L69

If your database was created before that date, it used the former defaults without them. But you can add them yourself, as you already found out yourself... :rocket:
